src/lipo.rs

```diff
@@ -37,13 +49,18 @@ pub(crate) fn build(cargo: &Cargo, meta: &Meta, targets: &[impl AsRef<str>]) ->
 
         output.push(&lib_name);
 
-        let mut cmd = Command::new("lipo");
-        cmd.arg("-create").arg("-output").arg(output);
-        cmd.args(inputs);
+        if let Ok(true) = is_output_updated(&output, &inputs) {
+            info!("Universal library for {} is updated", package.name());
+        }
+        else {
```

Comment:
I think the code can be simplified a bit by adding a `continue` at the end of the `if` block. That would make the `else` unnecessary.